docs/src/CommutativeAlgebra/GroebnerBases/orderings.md

```diff
@@ -159,14 +159,24 @@ $x^\alpha > x^\beta \;  \Leftrightarrow \;  \deg(x^\alpha) > \deg(x^\beta)  \;\t
 deglex(R::MPolyRing)
 ```
 
-#### The Reverse Lexicographical Ordering
+#### The Inverse Lexicographical Ordering
 
-The *reverse lexicographical ordering* `revlex` is defined by setting
+The *inverse lexicographical ordering* `invlex` is defined by setting
 
 $x^\alpha > x^\beta \;  \Leftrightarrow \;\exists \; 1 \leq i \leq n: \alpha_n = \beta_n, \dots, \alpha_{i+1} = \beta_{i+1}, \alpha_i  > \beta_i.$
 
 ```@docs
-revlex(R::MPolyRing)
+invlex(R::MPolyRing)
+```
+
+#### The Degree Inverse Lexicographical Ordering
+
+The *degree inverse lexicographical ordering* `deginvlex` is defined by setting
+
+$x^\alpha > x^\beta \;  \Leftrightarrow \deg(x^\alpha) > \deg(x^\beta)  \;\text{ or } \;\exists \; 1 \leq i \leq n: \alpha_n = \beta_n, \dots, \alpha_{i+1} = \beta_{i+1}, \alpha_i  > \beta_i.$
```

Comment:
This and all other definitions of `deg` or weighted orderings are wrong due to the "or". There is always an `\deg(x^\alpha) = \deg(x^\beta) and` missing on the right-hand side of the "or".

Reply:
Should be resolved now.